tools/istio-docker.mk

```diff
@@ -314,23 +321,23 @@ docker.scan_images: $(DOCKER_PUSH_TARGETS)
 # Base image for 'debug' containers.
 # You can run it first to use local changes (or guarantee it is built from scratch)
 docker.basedebug:
-	docker build -t istionightly/base_debug -f docker/Dockerfile.xenial_debug docker/
+	docker build -t docker.io/sdake/base_debug:${VERSION} -f docker/Dockerfile.xenial_debug docker/
 
 # Run this target to generate images based on Bionic Ubuntu
 # This must be run as a first step, before the 'docker' step.
 docker.basedebug_bionic:
-	docker build -t istionightly/base_debug_bionic -f docker/Dockerfile.bionic_debug docker/
-	docker tag istionightly/base_debug_bionic istionightly/base_debug
+	docker build -t docker.io/sdake/base_debug_bionic:$(VERSION) -f docker/Dockerfile.bionic_debug docker/
+	docker tag docker.io/sdake/base_debug_bionic:$(VERSION) docker.io/sdake/base_debug:$(VERSION)
 
```

Comment:
I think this is a great idea to name these base_debug images different instead of how it was. Thanks.

Reply:
good thinking.  @howardjohn can you please make a recommendation on what the conical repo/name should be for the base image?  I am just going to sed my work prior to removal of the WIP state.

Reply:
What do you think of using tags? Like docker.io/istio/base:1.3-xenial? And for the image we actually use just docker.io/istio/base:1.3, the -xenial -bionic are just if others want to use them.

Or we remove the special variants.. does anyone use/test them?

Reply:
I want per z-stream bases.

Reply:
I don't know if anyone uses the variants.  I know they are not tested in automation.

Reply:
I'm fine with removing them, @costinm may disagree though. They certainly are never tested or published though so I doubt anyone is using them

Reply:
I'd suggest handling variant distros in a followup if need be.  I prefer to improve one thing at a time so that I am aware of the change was an actual improvement or not.  Changing multiple variables is hard to detect what variable resulted in the improvement in the system.

Reply:
marking resolved for now.  This thread was more about my personal docker ehub being used, but now a dev image is available - this part of the thread isn't relevant.  The second part of the thread involves variants of distros - please feel free to open that as a review comment separately.